### PR TITLE
[zh-cn] improve security

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by windsonsea — APPROVED** [review]
LGTM

File: content/zh/docs/ops/best-practices/security/index.md [hunk 1]
```diff
@@ -436,25 +447,26 @@ spec:
...
 
 * `Gateway` 控制了网关的端口和 TLS 设置。
 * `VirtualService` 控制了路由逻辑。虚拟服务通过 `Gateway` 资源的 `gateways` 的字段直接被引用。
    并且`Gateway` 和 `VirtualService` 资源中的 `hosts` 字段需保持一致。
 
-### 限制 `Gateway` 创建权限{#restrict-gateway-creation-privileges}
+### 限制 `Gateway` 创建权限 {#restrict-gateway-creation-privileges}
 
 Istio 推荐将网关资源创建权限只分配给信任的集群管理员。这可以通过
 [Kubernetes RBAC policies](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/rbac/)
```
Comment (wilsonwu) [inline content/zh/docs/ops/best-practices/security/index.md:460]:
```suggestion
[Kubernetes RBAC 策略](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/rbac/)
```

**Comment by ericvn** [comment]
```
./zh/docs/ops/best-practices/security/index.md:186: MD009 Trailing spaces
./zh/docs/ops/best-practices/security/index.md:24: MD012 Multiple consecutive blank lines

Further documentation is available for these failures:
 - MD009: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md009---trailing-spaces
 - MD012: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md012---multiple-consecutive-blank-lines
```

File: content/zh/docs/ops/best-practices/security/index.md [hunk 2]
```diff
@@ -11,15 +11,17 @@ Istio 安全功能提供强大的身份，强大的策略，透明的 TLS 加密
...
+默认情况下代理会工作在[宽容模式](/zh/docs/concepts/security/#permissive-mode)下，
 这意味着代理会允许双向 TLS 认证的流量以及纯文本流量。
 
-尽管对于渐进式配置或允许流量来自没有 Istio 代理的客户端来说，这个模式是必需的。这个模式也削弱了安全。
-因此，建议尽早[迁移到 strict 模式](/zh/docs/tasks/security/authentication/mtls-migration/)，
-为了强制在流量进行双向 TLS 认证。
+尽管对于渐进式配置或允许流量来自没有 Istio 代理的客户端来说，这个模式是必需的。但这个模式也削弱了安全性，
+因此，为了强制在流量进行双向 TLS 认证，
+应建议尽早[迁移到 strict 模式](/zh/docs/tasks/security/authentication/mtls-migration/)。
+
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
应建议尽早[迁移到 strict 模式](/zh/docs/tasks/security/authentication/mtls-migration/)。
```

File: content/zh/docs/ops/best-practices/security/index.md [hunk 3]
```diff
@@ -555,20 +569,21 @@ spec:
...
 
 Istio 可以[自动确定流量协议](/zh/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection)。
 但为了避免意外或者有意的误检测，从而导致意外流量行为发生。推荐[显式地声明协议](/zh/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection)。
 
-## CNI 网络容器接口{#CNI}
+## CNI 网络容器接口 {#CNI}
 
 为了透明地劫持所以流量， Istio 依赖 通过 `istio-init` `initContainer` 配置 `iptables` 规则。
 这增加了一个[要求](/zh/docs/ops/deployment/requirements/)，即需要提供给 Pod `NET_ADMIN`
 和 `NET_RAW` [capabilities](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container)。
```
Comment (wilsonwu) [inline content/zh/docs/ops/best-practices/security/index.md:580]:
```suggestion
和 `NET_RAW` [兼容性](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container)。
```

File: content/zh/docs/ops/best-practices/security/index.md [hunk 4]
```diff
@@ -555,20 +569,21 @@ spec:
...
 
-为了减少给予 Pods 的权限， Istio 提供了 [CNI plugin](/zh/docs/setup/additional-setup/cni/) 功能，即不再需要以上权限。
+为了减少给予 Pods 的权限， Istio 提供了 [CNI plugin](/zh/docs/setup/additional-setup/cni/) 功能，
```
Comment (wilsonwu) [inline content/zh/docs/ops/best-practices/security/index.md:582]:
```suggestion
为了减少给予 Pods 的权限， Istio 提供了 [CNI 插件](/zh/docs/setup/additional-setup/cni/)功能，
```

File: content/zh/docs/ops/best-practices/security/index.md [hunk 5]
```diff
@@ -169,17 +172,21 @@ Istio 授权策略能够基于 HTTP 请求的 URL 路径实现。
...
-这项配置声明在 [mesh config](/zh/docs/reference/config/istio.mesh.v1alpha1/) 的 [`pathNormalization`](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ProxyPathNormalization) 字段。
+这项配置声明在 [mesh config](/zh/docs/reference/config/istio.mesh.v1alpha1/)
+的 [`pathNormalization`](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ProxyPathNormalization)
+字段。
 {{< /tip >}}
 
 着重强调，规范化算法按照以下顺序执行：
 1. 百分位解码 `%2F`, `%2f`, `%5C` 和 `%5c`。
-1. [RFC 3986](https://tools.ietf.org/html/rfc3986) 以及其他由 Envoy[`normalize_path`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#envoy-v3-api-field-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-normalize-path) 选项实现的规范。
+1. [RFC 3986](https://tools.ietf.org/html/rfc3986) 以及其他由
```
Comment (ericvn) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
1. [RFC 3986](https://tools.ietf.org/html/rfc3986) 以及其他由
```

**Review by ericvn — COMMENTED** [review]
I think I have the lint items as suggestions.

**Pull request merged**